Library/Formula/proftpd.rb

```diff
@@ -1,18 +1,35 @@
 class Proftpd < Formula
   desc "Highly configurable GPL-licensed FTP server software"
   homepage "http://www.proftpd.org/"
-  url "ftp://ftp.proftpd.org/distrib/source/proftpd-1.3.4d.tar.gz"
-  sha256 "c198b53991ce641eae6b3237e856e59f0bfe8330794145b49cae33f85b6f5370"
-
-  # fixes unknown group 'nogroup'
-  patch :DATA
+  url "ftp://ftp.proftpd.org/distrib/source/proftpd-1.3.5a.tar.gz"
+  sha256 "a1f48df8539c414ec56e0cea63dcf4b8e16e606c05f10156f030a4a67fae5696"
 
   def install
+    # fixes unknown group 'nogroup'
+    inreplace "sample-configurations/basic.conf" do |s|
+      s.gsub! "Group				nogroup", "Group				nobody"
+    end
+
+    # fixes Operation not permitted
+    inreplace "Makefile.in" do |s|
+      s.gsub! /chown( -h)? \$\(INSTALL_USER\):\$\(INSTALL_GROUP\).*\n/, ""
+      s.gsub! /-o \$\(INSTALL_USER\) -g \$\(INSTALL_GROUP\) /, ""
+    end
+
+    inreplace "Make.rules.in" do |s|
+      s.gsub! /-o \$\(INSTALL_USER\) -g \$\(INSTALL_GROUP\) /, ""
+    end
+
+    inreplace "contrib/Makefile.in" do |s|
+      s.gsub! /-o \$\(INSTALL_USER\) -g \$\(INSTALL_GROUP\) /, ""
+    end
+
     system "./configure", "--prefix=#{prefix}",
                           "--sysconfdir=#{etc}",
                           "--localstatedir=#{var}"
     ENV.j1
-    system "make", "INSTALL_USER=`whoami`", "INSTALL_GROUP=admin", "install"
+
+    system "make", "install"
```

Comment:
Without the `inreplaces` above, the build will fail with a bunch of `Operation not permitted` errors since it will default to 'INSTALL_USER=root` and `INSTALL_GROUP=wheel`